server/Cargo.toml

```diff
@@ -4,7 +4,6 @@ description = "Assets Service REST server"
 edition     = "2021"                                                       # Allowed values are 2015, 2018, 2021
 homepage    = "https://www.arrowair.com/docs/documentation/services/intro"
```

Comment:
use homepage.workspace = true and update the toplevel workspace C

use homepage.workspace = true and update the toplevel workspace Cargo.toml, see svc-storage's PR for example

same for all of the other attributes aside from "name", "description", and "version"
argo.toml, see svc-storage's PR for example

same for all of the other attributes aside from "name", "description", and "version"

Reply:
[d227142](https://github.com/Arrow-air/svc-assets/pull/18/commits/d2271428712237b35575f5be1116dae5f0b0dd87)